nibabel/gifti/gifti.py

```diff
@@ -68,10 +68,10 @@ def print_summary(self):
 
 
 class GiftiNVPairs(object):
-
+    """
     name = str
     value = str
-
+    """
```

Comment:
Yes, I agree, it's good to document these.   I just looked that up in https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#class-docstring

So I think this should be:

```
class GiftiNVPairs(object):
    """Name-value pair objects

    Attributes
    -------------
    name : str
        Maybe some text.
    value   : str
        Might be text here too.
    """
```